src/com/seafile/seadroid2/SeadroidApplication.java

```diff
@@ -20,6 +21,15 @@ public void onCreate() {
         super.onCreate();
         SeadroidApplication.context = getApplicationContext();
         initImageLoader(getApplicationContext());
+
+        // set gesture lock if available
+        AppLockManager.getInstance().enableDefaultAppLockIfAvailable(this);
+        if (AppLockManager.getInstance().isAppLockFeatureEnabled()) {
```

Comment:
I leave this for future use, in order to disable the lock function to some specific activities.